packages/react/src/components/Box/Box.tsx

```diff
@@ -17,7 +17,8 @@ export interface BoxProps
     ChildrenComponentProps {}
 
 /**
- * A Box is a basic component (no default styles)
+ * A Box is an abstract layout component, is frequently used for slots in other Stardust components.
```

Comment:
![image](https://user-images.githubusercontent.com/14183168/54754110-6a0ac800-4beb-11e9-8637-43da8c952bac.png)

I moved this warning upper. I don't to remove existing description for component because we have an assertion for it in UTs:

![image](https://user-images.githubusercontent.com/14183168/54754197-9f171a80-4beb-11e9-8790-7d0cad8e7c6c.png)

Reply:
oh, ok, then this is an improvement anyway 👍